setup.py

```diff
@@ -65,4 +65,10 @@
             "dev": doc_requirements + test_requirements,
         },
         zip_safe=False,
+        use_scm_version={
+            "write_to": "src/vivarium_cluster_tools/_version.py",
+            "write_to_template": '__version__ = "{version}"\n',
```

Comment:
Is this supposed to be an f-string? Where does `{version}` come from?

Reply:
No, not an f-string here. This is for setuptools_ACM to use to populate _version.py at setup.